src/Generator.cpp

```diff
@@ -115,6 +115,26 @@ int generate_filter_main(int argc, char **argv, std::ostream &cerr) {
         cerr << kUsage;
         return 1;
     }
+    GeneratorBase::EmitOptions emit_options;
+    std::string emit_flag = flags_info["-e"];
+    while (emit_flag != "") {
+      size_t pos = emit_flag.find(",");
+      std::string opt = emit_flag.substr(0, pos);
+      if (opt == "assembly") {
+        emit_options.emit_assembly = true;
+      } else if (opt == "bitcode") {
+        emit_options.emit_bitcode = true;
+      } else if (opt == "stmt") {
+        emit_options.emit_stmt = true;
+      } else if (opt == "html") {
+        emit_options.emit_stmt_html = true;
+      }
```

Comment:
The usage should probably indicate that these are the options for emit, and there should probably be an else case here that triggers the usage message.